src/Discord.Net.Analyzers/GuildAccessAnalyzer.cs

```diff
@@ -1,19 +1,26 @@
-using System;
-using System.Collections.Immutable;
-using System.Linq;
+using Discord.Commands;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Diagnostics;
-using Discord.Commands;
+using System;
+using System.Collections.Immutable;
+using System.Linq;
 
 namespace Discord.Analyzers
 {
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     public sealed class GuildAccessAnalyzer : DiagnosticAnalyzer
     {
         private const string DiagnosticId = "DNET0001";
+
+/* Unmerged change from project 'Discord.Net.Analyzers(netstandard2.1)'
+Before:
         private const string Title = "Limit command to Guild contexts.";
+After:
+        private const string Title = "Limit command to Guild contexts";
+*/
+        private const string Title = "Limit command to Guild contexts";
```

Comment:
```suggestion
        private const string Title = "Limit command to Guild contexts.";
```
I think this shouldn't be changed

Reply:
I agree on this change, it changes the string, which shouldn't have happene.